slack.php

```diff
@@ -57,90 +69,115 @@
                     ],
                     [
                         'title' => 'Stage',
-                        'value' => $stage,
+                        'value' => get('stages')[0],
                         'short' => true,
                     ],
                     [
                         'title' => 'Branch',
-                        'value' => $branch,
+                        'value' => (get('branch') ? get('branch') : 'master'),
```

Comment:
1. The `get()` function has "default value" parameter
2. The `$branch` is readable.